src/providers/ldap/sdap_async_connection.c

```diff
@@ -1882,6 +1883,8 @@ static void sdap_cli_auth_step(struct tevent_req *req)
                             user_dn, authtok,
                             dp_opt_get_int(state->opts->basic,
                                            SDAP_OPT_TIMEOUT));
+    sss_authtok_wipe_password(authtok);
```

Comment:
If wonder if handling would be easier if destructors are added in authtok.c with would wipe the data if `talloc_free(authtok)` is called?

Reply:
Right, this is exactly the plan. I will prepend `authtok.c` patch before and amend this one.

Reply:
I added `authtok.c` patch and removed  explicit `wipe`.